SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -597,6 +625,10 @@ private void queueMail(final Collection<SingularityEmailDestination> destination
       }
     }
 
+    if (LOG_EMAILS_FOR_DEBUG) {
+      LOG.debug("TheMail: " + body);
+    }
+
```

Comment:
We should minimize the number of times the email body is logged -- i think just having this one here will suffice